app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardFragment.kt

```diff
@@ -52,4 +52,7 @@ class ConceptCardFragment : InjectableDialogFragment() {
     super.onStart()
     dialog?.window?.setWindowAnimations(R.style.FullScreenDialogStyle)
   }
+
+  fun getSkillId() = conceptCardPresenter.skillId
+  fun getExplanationTextView() = conceptCardPresenter.explanationTextView
```

Comment:
This dependency inversion doesn't quite fit to me. Could we pass the skill ID & text view into the view model, instead of retrieving it? The view model ought to not depend directly on the fragment, I think.

Reply:
Updated so that ViewModel is set with these variables before binding.